node_modules/oae-core/manageaccess/js/manageaccess.js

```diff
@@ -200,6 +200,9 @@ define(['jquery', 'underscore', 'oae.core', 'jquery.autosuggest'], function($, _
                     return createMemberProfileForEmail(email, invitationsUpdates[email]);
```

Comment:
In order to make more space for the email address, I would still be in favour of changing the `Resend invitation` button to an airplane icon. In mobile view, we can just put the email address at the top and resend icon - role - remove at the bottom.